src/services/mediaService.tsx

```diff
@@ -851,6 +860,19 @@ export class MediaService extends EventEmitter {
 		logger.debug('rtcStatsCloseCallback()');
 	}
 
+	public initMonitor(): void {
+		if (!edumeetConfig.observertc) {
+			return;
+		}
+		this.monitor = createClientMonitor(edumeetConfig.observertc);
```

Comment:
Would it be possible to lazy load this so that it is not included in the main bundle and only fetched if observer is actually used?